stable/kommander-thanos/values.yaml

```diff
@@ -0,0 +1,53 @@
+# Default values for kommander-thanos.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Internal address for the cluster's Thanos gRPC service.
+# thanosAddress: "HOST:PORT"
+thanosAddress: "prometheus-kubeaddons-prom-prometheus.kubeaddons.svc.cluster.local:10901"
+
+federate:
+  systemNamespace:
+    name: kommander-system
+
+thanos:
+  store:
+    enabled: false
+  compact:
+    enabled: false
+  bucket:
+    enabled: false
+  sidecar:
+    enabled: false
+
+  query:
+    # Name of HTTP request header used for dynamic prefixing of UI links and redirects.
+    webPrefixHeader: "X-Forwarded-Prefix"
+    # Enable DNS discovery for stores
+    storeDNSDiscovery: false
+    # Enable DNS discovery for sidecars (this is for the chart built-in sidecar service)
+    sidecarDNSDiscovery: false
+    # Addresses of statically configured store API servers (repeatable).
+    stores: []
+    # Add extra arguments to the compact service
+    extraArgs:
+    - "--grpc-client-tls-secure"
+    - "--grpc-client-tls-cert=/etc/certs/tls.crt"
+    - "--grpc-client-tls-key=/etc/certs/tls.key"
+    - "--grpc-client-tls-ca=/etc/certs/ca.crt"
+    - "--grpc-client-server-name=server.thanos.localhost.localdomain"
+    certSecretName: kommander-thanos-client-tls
+    http:
+      ingress:
+        enabled: true
```

Comment:
maybe default to `false` since we're not exposing this service out to users? it is useful for debugging though, so we can easily override the value when deploying kommander.

Reply:
Would we want to keep for customer support? We don't have to link to it anywhere, but it might be handy for us or the support team if we can browse to a URL and get a list of stores and their connection status, without needing to redeploy or reconfigure Kommander. I'm not sure if there's another way to get that information out of Querier.